src/App.js

```diff
@@ -1,15 +1,24 @@
 import React from 'react';
 import './App.css';
 import Login from "./pages/Login"
-import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom"
+import { BrowserRouter as Router, Switch, Route, Redirect } from "react-router-dom"
+
 
 
 
 function App() {
+  //const login = <Login></Login>
```

Comment:
This is dead code. If it is not in use, then you can remove it.

Reply:
ok, will delete then.